ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/CommandFactories.java

```diff
@@ -265,6 +268,29 @@ private static LogicalSchema buildSchema(final TableElements tableElements) {
       throw new KsqlException("The statement does not define any columns.");
     }
 
+    tableElements.forEach(e -> {
+      if (e.getName().equalsIgnoreCase(SchemaUtil.ROWTIME_NAME)) {
+        throw new KsqlException("'" + e.getName() + "' is a reserved field name.");
+      }
+
+      final boolean isRowKey = e.getName().toUpperCase().equals(SchemaUtil.ROWKEY_NAME);
```

Comment:
For consistency why not use equalsIgnoreCase like on line 272?

Reply:
Yep, will do. (This code was a cut & paste job, but it's a good call)